src/globus_cli/commands/flows/run/show_logs.py

```diff
@@ -84,6 +87,13 @@ def show_logs_command(
             entry_iterator, fields=fields, json_converter=entry_iterator.json_converter
         )
 
+    if run_doc["status"] == "INACTIVE":
+        print_command_hint(
+            "\nNOTE: This run is INACTIVE. "
+            "No further logs will be added until it is resumed.",
```

Comment:
I find the implicit string concatenation confusing in the presence of additional function arguments.

Would it be an option to enclose this in parentheses so the implicit string concatenation isn't indented to match the second function argument that appears on the third line?

Reply:
I just tried out my new alias for putting in a `Co-authored-by` line with this fix. Let me know if you'd rather squash it, but I've made the tweak.

My brain has been sufficiently altered by `black` at this point that if something is not a requirement enforced by the formatter, I'm not sure if I have a stylistic preference. :joy: 
Looking more carefully, I agree that the parens improve readability.

Reply:
Crud. Adding me as `Co-authored-by` runs afoul of my 'verify signatures' configuration.

![image](https://github.com/globus/globus-cli/assets/39996/ed8851a5-1031-4ee1-8bbf-4173004619c9)

I don't think there's a solution for this, but I appreciate the hat-tip!